index-pl.html

```diff
@@ -104,18 +104,18 @@ <h2>Say it, Don't Spray It</h2>
 
   <div class="container">
 
-    <p>Masks reduce the mouth spray traveling between people — by blocking or by redirecting the spray — thereby reducing the chance of infection.</p>
+    <p>Maski zmniejszają zasięg aerozolu z ust, który może dotrzeć do innych osób — poprzez zatrzymanie lub przekierowanie — i w rezultacie zmniejszają prawdopodobieństwo zakażenia.</p>
 
     <p>
-      It's worth keeping in mind that no mask is perfect.
-      Even the N95 masks recommended for health workers are only guaranteed to block <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% of the hardest-to-block particles</a> (and that’s only if you <a href="https://www.n95decon.org/publications">wear them correctly</a>).
+      Należy pamiętać, że nie ma idealnej maski.
+      Nawet najbardziej skuteczne maski (FFP2 lub FFP3), zalecane dla pracowników służby zdrowia, zapewniają zatrzymanie <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">94%-99% cząsteczek wirusowych</a> (dodadkowo tylko jeśli <a href="https://www.n95decon.org/publications">noszone prawidłowo</a>).
```

Comment:
"dodadkowo tylko jeśli noszone..." -> "i to tylko jeśli są noszone..."

Reply:
ponawiam

Reply:
fixed